docs/api.rst

```diff
@@ -155,7 +155,7 @@ Handling user creation, sign-in, log-out and querying
                   "historical_price_oracles": ["cryptocompare", "coingecko"],
                   "taxable_ledger_actions": ["income", "airdrop"],
                   "ssf_0graph_multiplier": 2,
-                  "non_sync_exchanges: [{"location": "binance", "name": "binance1"}]
+                  "non_sync_exchanges": [{"location": "binance", "name": "binance1"}]
```

Comment:
I tought we had validation for json in the docs. Could be a good thing to have

Reply:
we don't. But these errors appear in `make html` for docs. We just don't check it in the CI.